tasks/main.yml

```diff
@@ -14,6 +14,7 @@
     apply:
```

Comment:
i can fix this later, but `firewall.yml` should be renamed `ufw.yml` and `name: install and configure ufw`

Reply:
This was an easy change, I've done it quickly. I should have renamed every place where this was using the word "firewall" when this means "ufw".

Reply:
Awesome, thanks.
I'll run some tests later today.